openpype/tools/project_manager/project_manager/delegates.py

```diff
@@ -83,15 +83,18 @@ class NumberDelegate(QtWidgets.QStyledItemDelegate):
         decimals(int): How many decimal points can be used. Float will be used
             as value if is higher than 0.
     """
-    def __init__(self, minimum, maximum, decimals, *args, **kwargs):
+    def __init__(self, minimum, maximum, decimals, step, *args, **kwargs):
         super(NumberDelegate, self).__init__(*args, **kwargs)
         self.minimum = minimum
         self.maximum = maximum
         self.decimals = decimals
+        self.step = step
 
     def createEditor(self, parent, option, index):
         if self.decimals > 0:
             editor = DoubleSpinBoxScrollFixed(parent)
+            editor.setSingleStep(self.step)
```

Comment:
Step is set only for decimal and not for integers.

Reply:
True, but I set the `step` to be a `setSingleStep` only if the editor is considered to be a DoubleSpinBox which works with decimals only.